### PR TITLE
CODENVY-349 Rename workspace agent packaged war name

### PR DESCRIPTION
**Pull request by mkuznyetsov**
Changed workspace agent war name from "ide.war" to "wsagent.war"
@eivantsov We'll need to update docs for Che.

### CODE REVIEW

**Comment by mkuznyetsov** [comment]
@skabashnyuk @vparfonov @ashumilova review please

File: dashboard/package.json [hunk 1]
```diff
@@ -51,7 +51,7 @@
     "main-bower-files": "~2.9.0",
     "minimist": "^1.2.0",
     "ng-annotate-loader": "0.0.10",
-    "phantomjs": "^2.1.3",
+    "phantomjs": "^2.1.7",
```
Comment (mshaposhnik) [inline dashboard/package.json]: Misha zatypuv

File: assembly/assembly-machine-server/src/assembly/webapps/ROOT/WEB-INF/rewrite.config [hunk 2]
```diff
@@ -1 +1 @@
-RewriteRule ^/api/ext/(.*)$     /ide/ext/$1    [L]
+RewriteRule ^/wsagent/ext/(.*)$     /wsagent/ext/$1    [L]
```
Comment (skabashnyuk) [inline assembly/assembly-machine-server/src/assembly/webapps/ROOT/WEB-INF/rewrite.config]: Do we need rewrite rules?

File: assembly/assembly-main/src/assembly/webapps/ROOT/WEB-INF/rewrite.config [hunk 3]
```diff
@@ -1,3 +1,3 @@
-RewriteRule ^/api/ext/(.*)$     /ide/ext/$1    [L]
+RewriteRule ^/wsagent/ext/(.*)$     /wsagent/ext/$1    [L]
 RewriteRule ^/api/(.*)$     /ide/api/$1    [L]
```
Comment (skabashnyuk) [inline assembly/assembly-main/src/assembly/webapps/ROOT/WEB-INF/rewrite.config:2]: Do we need rewrite rules?

**Comment by skabashnyuk** [comment]
other ok

File: assembly/assembly-machine-server/src/assembly/webapps/ROOT/WEB-INF/rewrite.config (hunk 2 above)
Comment (mkuznyetsov) [inline assembly/assembly-machine-server/src/assembly/webapps/ROOT/WEB-INF/rewrite.config]: removed rule

File: assembly/assembly-main/src/assembly/webapps/ROOT/WEB-INF/rewrite.config (hunk 3 above)
Comment (mkuznyetsov) [inline assembly/assembly-main/src/assembly/webapps/ROOT/WEB-INF/rewrite.config:2]: removed rule

**Comment by skabashnyuk** [comment]
ok

**Pull request merged**